mono/metadata/threads.c

```diff
@@ -125,8 +126,8 @@ static void mono_threads_unlock (void);
 static MonoCoopMutex threads_mutex;
 
 /* Controls access to the 'joinable_threads' hash table */
-#define joinable_threads_lock() mono_os_mutex_lock (&joinable_threads_mutex)
-#define joinable_threads_unlock() mono_os_mutex_unlock (&joinable_threads_mutex)
+#define joinable_threads_lock() (mono_os_mutex_lock (&joinable_threads_mutex))
```

Comment:
Line damage.

Reply:
Fixing the code. Macros need parens. I made it a separate commit.